ui/app/src/services/dashboard.ts

```diff
@@ -109,5 +109,5 @@ export function fetchLegacyDeploymentHistory(
     ...(filterBy && { filterType: filterBy })
   });
 
-  return get(`/studio/api/1/services/api/1/deployment/get-deployment-history.json${qs}`).pipe(pluck('response'));
+  return get(`/studio/api/2/publish/history.json${qs}`).pipe(pluck('response'));
```

Comment:
Is API 2 returning same response format?

Reply:
it is responding in the same format as 3.1